jsonrpc/types.go

```diff
@@ -17,6 +17,8 @@ type transactionOrHash interface {
 type transaction struct {
 	Nonce       argUint64      `json:"nonce"`
 	GasPrice    argBig         `json:"gasPrice"`
+	GasTipCap   *argBig        `json:"gasTipCap,omitempty"`
+	GasFeeCap   *argBig        `json:"gasFeeCap,omitempty"`
```

Comment:
Are these fields pointers so that they can accept `nil` values?

Reply:
Yea, there is a test to check it because they are optional. Addressed there: https://github.com/0xPolygon/polygon-edge/pull/1210#discussion_r1109764684